rtdata/languages/default

```diff
@@ -61,6 +61,7 @@ EXIFFILTER_IMAGETYPE;Image type
 EXIFFILTER_ISO;ISO
 EXIFFILTER_LENS;Lens
 EXIFFILTER_METADATAFILTER;Enable metadata filters
+EXIFFILTER_PATH;Path
 EXIFFILTER_SHUTTER;Shutter
```

Comment:
I'm concerned that "Path" is a little vague. Something like "File path" is more clear.